pipelines/managed/release-to-github/README.md

```diff
@@ -24,6 +24,7 @@ Tekton release pipeline to release binaries extracted from the image built with
 ## Changes in 4.6.0
 * Pass taskGitUrl and taskGitRevision to extract-binaries-from-image task
 * Pass taskGitUrl and taskGitRevision to base64-encode-checksum task
+* Pass taskGitUrl and taskGitRevision to create-github-release task
```

Comment:
Should be a new version? Or you wanna treat all of these as one update? Fine with me.